Tests/SonarScanner.MSBuild.PreProcessor.Test/SonarWebServerTest.cs

```diff
@@ -40,24 +39,19 @@ public class SonarWebServerTest
 {
     private const string ProjectKey = "project-key";
 
-    private readonly IDownloader downloader;
-    private readonly IJreCache jreCache;
-    private readonly TestLogger logger;
-    private readonly Version version;
+    private readonly TestLogger logger = new();
```

Comment:
Same here, the logger is reused so the state is not clear for each test. Initialization should be done in the Init method.